scrape_your_plate.py

```diff
@@ -88,7 +89,7 @@ def get_recipe(session, id):
         r = requests.get(thumb['src'])
 
         m = re.search('recipes/(.+\.jpg)', thumb['src'])
-        with open('./recipes/img/{}'.format(m.group(1)),'wb') as img:
+        with open(imgpath + '/{}'.format(m.group(1)),'wb') as img:
```

Comment:
Using os.path.join further down was a good call. I think you'd want to use it here too.